objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -92,10 +92,27 @@ public void handleMessage(ExtensionServiceMessage message) {
                             "Request must be a map", null);
                 }
                 
-                // Read, process, and send the image
-                ImageRetrieverResults data = source.retrieveImage(message);
-                if (data != null) {
-                    source.sendDataFromImage(data, message);
+                Map<String, ?> request = (Map<String, ?>) message.getObject();
+                String replyAddress = ExtensionServiceMessage.extractReplyAddress(message);
+                if (request.get("operation") instanceof String) {
+                    String operation = (String) request.get("operation");
+                    if (operation.equalsIgnoreCase("upload")) {
+                        source.uploadLocalImages(request, replyAddress);
+                    } else if (operation.equalsIgnoreCase("delete")) {
+                        source.deleteLocalImages(request, replyAddress);
+                    } else {
```

Comment:
Suppose I send an operation of "nutzo".  I think it will default to something here instead of returning in invalid operation -- which, I think, it should do.